examples/boto3/README.md

```diff
@@ -0,0 +1,14 @@
+# Introduction
+This directory contains examples on how to use boto3 to execrsize thew RadosGW extensions to the S3 API supported by AWS.
+
+# Users
+For the standard boto3 client to support these extensions, the: ``service-2.sdk-extras.json`` file should be placed under: ``~/.aws/models/s3/2006-03-01/`` directory.
+For more information see [here](https://github.com/boto/botocore/blob/develop/botocore/loaders.py#L33).
+
+# Developers
+Anyone developing an extension to the S3 API supported by AWS, please modify ``service-2.sdk-extras.json`` (all extensions should go into the same file), so that boto3 could be used to test the new API. 
+In addition, python files with code samples should be added to this directory demonstrating use of the new API.
+When testing you changes please:
+- make sure that the modified file is in the boto3 path as explained above
+- make sure that the standard S3 tests suit is not broken, even with the extensions files in the path
+
```

Comment:
In addition to boto, this allows aws cli to use these commands as well. Please make sure to add a useful help text while adding the json fields as these will be directly used by the cli.

Reply:
In examples a simple example like 

`aws s3api list-objects --bucket=bucket1 --allow-unordered`

Reply:
will test that for the different extensions and update

Reply:
@theanalyst do you think we should rename the directory, or would "boto3" make sense in both the cases?

Reply:
I think it should be fine

Reply:
added note and 3 examples regarding "aws cli" in the readme